lib/src/ui/user_account_layout.dart

```diff
@@ -24,14 +26,24 @@ import 'user_account_view_version.dart';
 
 class UserAccountLayout extends StatelessWidget {
   static const num _cardMarginTop = 2.25;
-  
+
+  static UpvotyStyle style = UpvotyStyle();
+  static Upvoty upvoty = Upvoty(style: style, pressBack: (context) {
+    Navigator.of(context).pop();
+    UserAccount(
+        referalCode: "\$0000T",
+        logout: () => print('logout'),
+        combinedKeys: 'teste')
+        .open(context);
+  });
+
   const UserAccountLayout({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    UserAccountService service =
-        Provider.of<UserAccountService>(context);
+    UserAccountService service = Provider.of<UserAccountService>(context);
```

Comment:
upvoty could be built here instead of line 31, as you have service and context available here.